v03_pipeline/lib/tasks/reference_data/base_updated_reference_dataset.py

```diff
@@ -0,0 +1,42 @@
+import hail as hl
+import luigi
+
+from luigi_pipeline.lib.hail_tasks import GCSorLocalTarget
+from v03_pipeline.lib.misc.io import does_file_exist
+from v03_pipeline.lib.paths import valid_reference_dataset_path
+from v03_pipeline.lib.reference_data.reference_dataset import ReferenceDataset
+from v03_pipeline.lib.tasks.base.base_loading_run_params import BaseLoadingRunParams
+from v03_pipeline.lib.tasks.base.base_write import BaseWriteTask
+
+
+@luigi.util.inherits(BaseLoadingRunParams)
+class UpdatedReferenceDataset(BaseWriteTask):
+    reference_dataset: ReferenceDataset
+
+    def output(self):
+        return GCSorLocalTarget(
+            valid_reference_dataset_path(
+                self.reference_genome,
+                self.reference_dataset,
+            ),
+        )
+
+    def create_table(self):
+        if does_file_exist(self.output().path):
```

Comment:
I think we don't need this anymore with `luigi` doing the `output` checking right?